src/system/RoutesModule/Api/Base/CacheApi.php

```diff
@@ -14,14 +14,14 @@
 
 use ModUtil;
 use UserUtil;
-use Zikula_AbstractApi;
+use Zikula_AbstractBase;
 use Zikula_View;
 use Zikula_View_Theme;
 
 /**
  * Cache api base class.
  */
-class CacheApi extends Zikula_AbstractApi
+class CacheApi extends Zikula_AbstractBase
```

Comment:
I still think this is a wrong use of `AbstractBase`, by the way. I don't think this class was ever intended to be used outside of being extended by `Zikula_AbstractApi` and `Zikula_AbstractController` (both obviously legacy classes). I strong recommend discontinuing its use outside the core and refactoring to a service class. If you must use a legacy class, use `Zikula_AbstractApi` directly.